website/source/docs/auth/gcp.html.md

```diff
@@ -127,8 +153,8 @@ management tool.
     ```text
     $ vault write auth/gcp/role/my-gce-role \
         type="gce" \
-        project_id="my-project" \
         policies="dev,prod" \
+        bound_projects="my-project1, my-project2" \
```

Comment:
Does it correctly handle spaces like this? I think Vault will interpret this as `<space>my-project2`